python/src/array.cpp

```diff
@@ -181,37 +287,43 @@ array array_from_list(T pl, std::optional<Dtype> dtype) {
   // Validate the shape and type
   auto type = validate_shape(pl, shape, 0);
 
-  size_t size = 1;
-  for (auto s : shape) {
-    size *= s;
```

Comment:
For some reason the `size` here is never used, so I deleted this block.

Reply:
Oops, good catch.